testsuite/cluster-test/Cargo.toml

```diff
@@ -46,5 +46,5 @@ futures = "0.3.5"
 tokio = { version = "0.2.21", features = ["full"] }
 async-trait = "0.1.31"
 
-kube = "0.32.0"
-k8s-openapi = { version = "0.7.1", default-features = false, features = ["v1_15"] }
+kube = "0.34.0"
+k8s-openapi = { version = "0.8.0", default-features = false, features = ["v1_15"] }
```

Comment:
It seems when their is an open k8s-openapi update (which will tend to fail due to the features section here), then kube will be unable to upgrade.